.github/workflows/full-check.yml

```diff
@@ -51,7 +51,9 @@ jobs:
            sudo make install
     - name: run_program
       run: |
-           dcfldd if=.github/workflows/test.txt of=/tmp/test2.txt hash=md5,sha1,sha256,sha384,sha512 2>&1 | grep bc0e4b74695142e0a0bdae87aea310d7078866cb
+           set -x
+           dcfldd if=.github/workflows/test.txt of=/tmp/test2.txt hash=md5,sha1,sha256,sha384,sha512 2>.github/workflows/actual-stderr.txt
+           diff -u .github/workflows/{actual,expected}-stderr.txt
```

Comment:
This looks a bit fragile to me, depending too much on the output formatting.

Reply:
@davidpolverari yes, it will break, whenenever the output format changes, but when that happens, it will go through a pull request and the related pull request will then also adjust the `.github/workflows/expected-stderr.txt` and that will even help that pull request to be very visual. So CI ensures that output and that file remain in sync and hence I would argue its "fragile with full control and benefits" rather than "just fragile". What do you think?

Reply:
@davidpolverari I see more work but no additional gains there. Sure we can grep the output for 5 different checksums in a loop but why? Complexity is higher and comparison to the actual output is as visual as it gets as a bonus side effect. And the output is apparently stable (not?). I really see no reasons to do it different with status quo, and I'm not shy of shell complexity [where needed and paying off](https://github.com/hartwork/wnpp.debian.net/blob/81cf297886aaf97334b2076829c2063b8f4613fa/docker/Dockerfile#L34-L36), just not here.

Reply:
Fair enough. We can do it that way then. We can change later _if_ the output changes very often.

Reply:
@davidpolverari thank you! :pray: